### DISCUSSION TITLE
Is searx.priv.pw caching requests?

### DISCUSSION THREAD

**Discussion by ononoki1** [searxng/searx-instances#140] [General]
Hi @vvaadim

I noticed your instance has a very low response time on searx.space (0.032s).

![](https://user-images.githubusercontent.com/82757890/180992684-5355dbee-86af-4160-90a4-9200880f8791.PNG)

Then I made requests via curl (`curl -I https://searx.priv.pw/search?q=test`) and found the `server-timing` header is exactly the same for every request (`server-timing: total;dur=866.878, render;dur=93.941, total_0_wikipedia;dur=225.956, total_1_bing;dur=406.301, total_2_google;dur=759.098, load_0_wikipedia;dur=222.546, load_1_bing;dur=385.66, load_2_google;dur=700.7`), which is hardly possible in normal case.

Caching requests is not privacy-friendly, and it also violates the promise you made when adding instance (see here: https://github.com/searxng/searx-instances/issues/67#issue-1251728067). Of course I might be wrong, but would you like to give an explanation?

**Comment by vvaadim** [comment]
Hey, you are right. I was experimenting with Nginx caching to speed up my instance a bit, no logs were retained eitherway and mostly static objects were being cached. Thanks for letting me know, I did not know that it was an issue. I have removed it. Sorry,
